masonry/src/core/properties.rs

```diff
@@ -0,0 +1,124 @@
+// Copyright 2025 the Xilem Authors
+// SPDX-License-Identifier: Apache-2.0
+
+use anymap3::{AnyMap, Entry};
+
+// TODO - Add PropertyValue wrapper struct that implements receiver trait.
+// Return PropertyValue<T> instead of Option<T> from methods.
```

Comment:
Why?

Reply:
The idea is that widget methods will use a lot of properties. Instead of having one `if let Some(foo) = ...` block per property access, which is three lines minimum with standard rustfmt, they should instead call methods directly on the optional property, for which we need something implementing the receiver trait.